Automattic/newspack-theme
# chore: update dependencies; add dependabot

### All Submissions:






### Changes proposed in this Pull Request:



Updates NPM packages; specifies Node support at v16; implements Dependabot.

### How to test the changes in this Pull Request:

Switch to Node v16 using NVM and ensure you can run `npm ci`, `npm run build`, `npm run dev`, and `npm run lint`.

No functional changes; a smoke test should suffice. Maybe pay special attention to the SCSS changes in 
ee1f449, which were the only manual code changes needed to suppress linting errors stemming from the updated rulesets (the JS changes were done by eslint, so we shouldn't need to test those).

### Other information:
